packages/wrangler/src/__tests__/tail.test.ts

```diff
@@ -712,6 +802,15 @@ function mockWebsocketAPIs(
 			const message = await api.ws.nextMessage;
 			return JSON.parse(message as string);
 		},
+		/**
+		 * Close the mock websocket and clean up the API.
+		 * The setTimeout forces a cycle to allow for closing and cleanup
+		 * @returns a Promise that resolves when the websocket is closed
+		 */
+		async closeHelper() {
+			api.ws.close();
+			await new Promise((resolve) => setTimeout(resolve, 0));
+		},
```

Comment:
It looks like this was working without the `setTimeout` before—why is this needed now? Would `process.nextTick` work?